scapy/layers/vxlan.py

```diff
@@ -85,7 +85,6 @@ def mysummary(self):
 # By default, set both ports to the RFC standard
 bind_layers(UDP, VXLAN, sport=4789, dport=4789)
 
-bind_layers(VXLAN, Ether)
```

Comment:
We should keep at least this:
```suggestion
bind_layers(VXLAN, Ether, NextProtocol=0)
```

Reply:
Ok, I'll add some tests for the fix, I hope to have it ready by tomorrow